code/datums/gamemode/objectives/target/assassinate.dm

```diff
@@ -44,8 +53,47 @@
 		return TRUE
 	//Borgs/brains/AIs count as dead for traitor objectives. --NeoFite
 	// If they're in an away mission/custom z-level by the time you're checking this might as well count them as MIA
+	switch (syndicate_checked)
```

Comment:
Doesn't the way you're checking this make it so that the objective is still considered "complete" on the round end scoreboard, even if you don't actually do the photo?

Reply:
Yep, that's intended. The photo is just a way to "lock" the result mid-round (and also enable getting the next target, which you might want to do since you don't know how many other agents there are)

Reply:
Makes sense.